### ISSUE TITLE
[BUG] Get a notification hub by the hub path not working

### ISSUE THREAD

**Issue by cepxy** [Azure/azure-notificationhubs-java-backend#169]
**Describe the bug**
Gets a notification hub by the hub path suddenly stopped working.

Our application suddenly broke a few days ago. The sdk is suddenly not able get a notification hub by the hub path.
No changes done on our side on our code or our azure services.

We were still using NotificationHubs.0.0.5, so I upgraded to latest Notification-Hubs-java-sdk.1.0.4 but that has the same issue.

***Exception or Stack Trace***
```
17:02:44.562 ERROR [httpclient-dispatch-1 ]                       digester3.Digester - End event threw exception
java.lang.reflect.InvocationTargetException: null
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_345]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_345]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_345]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_345]
	at org.apache.commons.beanutils.MethodUtils.invokeMethod(MethodUtils.java:278) ~[commons-beanutils-1.9.4.jar:1.9.4]
	at org.apache.commons.digester3.CallMethodRule.end(CallMethodRule.java:500) ~[commons-digester3-3.2.jar:3.2]
	at org.apache.commons.digester3.Digester.endElement(Digester.java:1128) ~[commons-digester3-3.2.jar:3.2]
	at com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.endElement(AbstractSAXParser.java:609) ~[?:1.8.0_345]
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanEndElement(XMLDocumentFragmentScannerImpl.java:1781) ~[?:1.8.0_345]
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl$FragmentContentDriver.next(XMLDocumentFragmentScannerImpl.java:2966) ~[?:1.8.0_345]
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl.next(XMLDocumentScannerImpl.java:601) ~[?:1.8.0_345]
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanDocument(XMLDocumentFragmentScannerImpl.java:504) ~[?:1.8.0_345]
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:841) ~[?:1.8.0_345]
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:770) ~[?:1.8.0_345]
	at com.sun.org.apache.xerces.internal.parsers.XMLParser.parse(XMLParser.java:141) ~[?:1.8.0_345]
	at com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.parse(AbstractSAXParser.java:1213) ~[?:1.8.0_345]
	at com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl$JAXPSAXParser.parse(SAXParserImpl.java:642) ~[?:1.8.0_345]
	at org.apache.commons.digester3.Digester.parse(Digester.java:1642) ~[commons-digester3-3.2.jar:3.2]
	at org.apache.commons.digester3.Digester.parse(Digester.java:1701) ~[commons-digester3-3.2.jar:3.2]
	at com.windowsazure.messaging.NotificationHubDescription.parseOne(NotificationHubDescription.java:170) ~[Notification-Hubs-java-sdk-1.0.4.jar:1.0.4]
	at com.windowsazure.messaging.NamespaceManager.lambda$getNotificationHubAsync$0(NamespaceManager.java:72) ~[Notification-Hubs-java-sdk-1.0.4.jar:1.0.4]
	at com.windowsazure.messaging.NotificationHubsService$1.completed(NotificationHubsService.java:73) [Notification-Hubs-java-sdk-1.0.4.jar:1.0.4]
	at com.windowsazure.messaging.NotificationHubsService$1.completed(NotificationHubsService.java:63) [Notification-Hubs-java-sdk-1.0.4.jar:1.0.4]
	at org.apache.hc.core5.concurrent.BasicFuture.completed(BasicFuture.java:123) [httpcore5-5.2.jar:5.2]
	at org.apache.hc.core5.concurrent.ComplexFuture.completed(ComplexFuture.java:72) [httpcore5-5.2.jar:5.2]
	at org.apache.hc.client5.http.impl.async.InternalAbstractHttpAsyncClient$1$1.completed(InternalAbstractHttpAsyncClient.java:280) [httpclient5-5.2.jar:5.2]
	at org.apache.hc.core5.http.nio.support.AbstractAsyncResponseConsumer$1.completed(AbstractAsyncResponseConsumer.java:101) [httpcore5-5.2.jar:5.2]
	at org.apache.hc.core5.http.nio.entity.AbstractBinAsyncEntityConsumer.completed(AbstractBinAsyncEntityConsumer.java:84) [httpcore5-5.2.jar:5.2]
	at org.apache.hc.core5.http.nio.entity.AbstractBinDataConsumer.streamEnd(AbstractBinDataConsumer.java:81) [httpcore5-5.2.jar:5.2]
	at org.apache.hc.core5.http.nio.support.AbstractAsyncResponseConsumer.streamEnd(AbstractAsyncResponseConsumer.java:142) [httpcore5-5.2.jar:5.2]
	at org.apache.hc.client5.http.impl.async.H2AsyncMainClientExec$1.streamEnd(H2AsyncMainClientExec.java:189) [httpclient5-5.2.jar:5.2]
	at org.apache.hc.core5.http2.impl.nio.ClientH2StreamHandler.consumeData(ClientH2StreamHandler.java:239) [httpcore5-h2-5.2.jar:5.2]
	at org.apache.hc.core5.http2.impl.nio.AbstractH2StreamMultiplexer$H2Stream.consumeData(AbstractH2StreamMultiplexer.java:1636) [httpcore5-h2-5.2.jar:5.2]
	at org.apache.hc.core5.http2.impl.nio.AbstractH2StreamMultiplexer.consumeDataFrame(AbstractH2StreamMultiplexer.java:1045) [httpcore5-h2-5.2.jar:5.2]
	at org.apache.hc.core5.http2.impl.nio.AbstractH2StreamMultiplexer.consumeFrame(AbstractH2StreamMultiplexer.java:734) [httpcore5-h2-5.2.jar:5.2]
	at org.apache.hc.core5.http2.impl.nio.AbstractH2StreamMultiplexer.onInput(AbstractH2StreamMultiplexer.java:445) [httpcore5-h2-5.2.jar:5.2]
	at org.apache.hc.core5.http2.impl.nio.AbstractH2IOEventHandler.inputReady(AbstractH2IOEventHandler.java:65) [httpcore5-h2-5.2.jar:5.2]
	at org.apache.hc.core5.http2.impl.nio.ClientH2IOEventHandler.inputReady(ClientH2IOEventHandler.java:39) [httpcore5-h2-5.2.jar:5.2]
	at org.apache.hc.core5.reactor.ssl.SSLIOSession.decryptData(SSLIOSession.java:600) [httpcore5-5.2.jar:5.2]
	at org.apache.hc.core5.reactor.ssl.SSLIOSession.access$200(SSLIOSession.java:74) [httpcore5-5.2.jar:5.2]
	at org.apache.hc.core5.reactor.ssl.SSLIOSession$1.inputReady(SSLIOSession.java:202) [httpcore5-5.2.jar:5.2]
	at org.apache.hc.core5.reactor.InternalDataChannel.onIOEvent(InternalDataChannel.java:142) [httpcore5-5.2.jar:5.2]
	at org.apache.hc.core5.reactor.InternalChannel.handleIOEvent(InternalChannel.java:51) [httpcore5-5.2.jar:5.2]
	at org.apache.hc.core5.reactor.SingleCoreIOReactor.processEvents(SingleCoreIOReactor.java:178) [httpcore5-5.2.jar:5.2]
	at org.apache.hc.core5.reactor.SingleCoreIOReactor.doExecute(SingleCoreIOReactor.java:127) [httpcore5-5.2.jar:5.2]
	at org.apache.hc.core5.reactor.AbstractSingleCoreIOReactor.execute(AbstractSingleCoreIOReactor.java:86) [httpcore5-5.2.jar:5.2]
	at org.apache.hc.core5.reactor.IOReactorWorker.run(IOReactorWorker.java:44) [httpcore5-5.2.jar:5.2]
	at java.lang.Thread.run(Thread.java:750) [?:1.8.0_345]
Caused by: java.lang.NoSuchMethodException: com.windowsazure.messaging.ApnsCredential.setkeyId(java.lang.String)
	at java.lang.Class.getMethod(Class.java:1786) ~[?:1.8.0_345]
	at com.windowsazure.messaging.PnsCredential.setProperty(PnsCredential.java:23) ~[Notification-Hubs-java-sdk-1.0.4.jar:1.0.4]
	... 48 more
17:02:44.592 ERROR [httpclient-dispatch-1 ]                       digester3.Digester - An error occurred while parsing XML from '(already loaded from stream)', see nested exceptions
org.xml.sax.SAXParseException: Error at line 1 char 840: com.windowsazure.messaging.ApnsCredential.setkeyId(java.lang.String)
	at org.apache.commons.digester3.Digester.createSAXException(Digester.java:3118) ~[commons-digester3-3.2.jar:3.2]
	at org.apache.commons.digester3.Digester.createSAXException(Digester.java:3146) ~[commons-digester3-3.2.jar:3.2]
	at org.apache.commons.digester3.Digester.endElement(Digester.java:1133) ~[commons-digester3-3.2.jar:3.2]
	at com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.endElement(AbstractSAXParser.java:609) ~[?:1.8.0_345]
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanEndElement(XMLDocumentFragmentScannerImpl.java:1781) ~[?:1.8.0_345]
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl$FragmentContentDriver.next(XMLDocumentFragmentScannerImpl.java:2966) ~[?:1.8.0_345]
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl.next(XMLDocumentScannerImpl.java:601) ~[?:1.8.0_345]
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanDocument(XMLDocumentFragmentScannerImpl.java:504) ~[?:1.8.0_345]
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:841) ~[?:1.8.0_345]
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:770) ~[?:1.8.0_345]
	at com.sun.org.apache.xerces.internal.parsers.XMLParser.parse(XMLParser.java:141) ~[?:1.8.0_345]
	at com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.parse(AbstractSAXParser.java:1213) ~[?:1.8.0_345]
	at com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl$JAXPSAXParser.parse(SAXParserImpl.java:642) ~[?:1.8.0_345]
	at org.apache.commons.digester3.Digester.parse(Digester.java:1642) ~[commons-digester3-3.2.jar:3.2]
	at org.apache.commons.digester3.Digester.parse(Digester.java:1701) ~[commons-digester3-3.2.jar:3.2]
	at com.windowsazure.messaging.NotificationHubDescription.parseOne(NotificationHubDescription.java:170) ~[Notification-Hubs-java-sdk-1.0.4.jar:1.0.4]
	at com.windowsazure.messaging.NamespaceManager.lambda$getNotificationHubAsync$0(NamespaceManager.java:72) ~[Notification-Hubs-java-sdk-1.0.4.jar:1.0.4]
	at com.windowsazure.messaging.NotificationHubsService$1.completed(NotificationHubsService.java:73) [Notification-Hubs-java-sdk-1.0.4.jar:1.0.4]
	at com.windowsazure.messaging.NotificationHubsService$1.completed(NotificationHubsService.java:63) [Notification-Hubs-java-sdk-1.0.4.jar:1.0.4]
	at org.apache.hc.core5.concurrent.BasicFuture.completed(BasicFuture.java:123) [httpcore5-5.2.jar:5.2]
	at org.apache.hc.core5.concurrent.ComplexFuture.completed(ComplexFuture.java:72) [httpcore5-5.2.jar:5.2]
	at org.apache.hc.client5.http.impl.async.InternalAbstractHttpAsyncClient$1$1.completed(InternalAbstractHttpAsyncClient.java:280) [httpclient5-5.2.jar:5.2]
	at org.apache.hc.core5.http.nio.support.AbstractAsyncResponseConsumer$1.completed(AbstractAsyncResponseConsumer.java:101) [httpcore5-5.2.jar:5.2]
	at org.apache.hc.core5.http.nio.entity.AbstractBinAsyncEntityConsumer.completed(AbstractBinAsyncEntityConsumer.java:84) [httpcore5-5.2.jar:5.2]
	at org.apache.hc.core5.http.nio.entity.AbstractBinDataConsumer.streamEnd(AbstractBinDataConsumer.java:81) [httpcore5-5.2.jar:5.2]
	at org.apache.hc.core5.http.nio.support.AbstractAsyncResponseConsumer.streamEnd(AbstractAsyncResponseConsumer.java:142) [httpcore5-5.2.jar:5.2]
	at org.apache.hc.client5.http.impl.async.H2AsyncMainClientExec$1.streamEnd(H2AsyncMainClientExec.java:189) [httpclient5-5.2.jar:5.2]
	at org.apache.hc.core5.http2.impl.nio.ClientH2StreamHandler.consumeData(ClientH2StreamHandler.java:239) [httpcore5-h2-5.2.jar:5.2]
	at org.apache.hc.core5.http2.impl.nio.AbstractH2StreamMultiplexer$H2Stream.consumeData(AbstractH2StreamMultiplexer.java:1636) [httpcore5-h2-5.2.jar:5.2]
	at org.apache.hc.core5.http2.impl.nio.AbstractH2StreamMultiplexer.consumeDataFrame(AbstractH2StreamMultiplexer.java:1045) [httpcore5-h2-5.2.jar:5.2]
	at org.apache.hc.core5.http2.impl.nio.AbstractH2StreamMultiplexer.consumeFrame(AbstractH2StreamMultiplexer.java:734) [httpcore5-h2-5.2.jar:5.2]
	at org.apache.hc.core5.http2.impl.nio.AbstractH2StreamMultiplexer.onInput(AbstractH2StreamMultiplexer.java:445) [httpcore5-h2-5.2.jar:5.2]
	at org.apache.hc.core5.http2.impl.nio.AbstractH2IOEventHandler.inputReady(AbstractH2IOEventHandler.java:65) [httpcore5-h2-5.2.jar:5.2]
	at org.apache.hc.core5.http2.impl.nio.ClientH2IOEventHandler.inputReady(ClientH2IOEventHandler.java:39) [httpcore5-h2-5.2.jar:5.2]
	at org.apache.hc.core5.reactor.ssl.SSLIOSession.decryptData(SSLIOSession.java:600) [httpcore5-5.2.jar:5.2]
	at org.apache.hc.core5.reactor.ssl.SSLIOSession.access$200(SSLIOSession.java:74) [httpcore5-5.2.jar:5.2]
	at org.apache.hc.core5.reactor.ssl.SSLIOSession$1.inputReady(SSLIOSession.java:202) [httpcore5-5.2.jar:5.2]
	at org.apache.hc.core5.reactor.InternalDataChannel.onIOEvent(InternalDataChannel.java:142) [httpcore5-5.2.jar:5.2]
	at org.apache.hc.core5.reactor.InternalChannel.handleIOEvent(InternalChannel.java:51) [httpcore5-5.2.jar:5.2]
	at org.apache.hc.core5.reactor.SingleCoreIOReactor.processEvents(SingleCoreIOReactor.java:178) [httpcore5-5.2.jar:5.2]
	at org.apache.hc.core5.reactor.SingleCoreIOReactor.doExecute(SingleCoreIOReactor.java:127) [httpcore5-5.2.jar:5.2]
	at org.apache.hc.core5.reactor.AbstractSingleCoreIOReactor.execute(AbstractSingleCoreIOReactor.java:86) [httpcore5-5.2.jar:5.2]
	at org.apache.hc.core5.reactor.IOReactorWorker.run(IOReactorWorker.java:44) [httpcore5-5.2.jar:5.2]
	at java.lang.Thread.run(Thread.java:750) [?:1.8.0_345]
Caused by: java.lang.NoSuchMethodException: com.windowsazure.messaging.ApnsCredential.setkeyId(java.lang.String)
	at java.lang.Class.getMethod(Class.java:1786) ~[?:1.8.0_345]
	at com.windowsazure.messaging.PnsCredential.setProperty(PnsCredential.java:23) ~[Notification-Hubs-java-sdk-1.0.4.jar:1.0.4]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_345]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_345]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_345]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_345]
	at org.apache.commons.beanutils.MethodUtils.invokeMethod(MethodUtils.java:278) ~[commons-beanutils-1.9.4.jar:1.9.4]
	at org.apache.commons.digester3.CallMethodRule.end(CallMethodRule.java:500) ~[commons-digester3-3.2.jar:3.2]
	at org.apache.commons.digester3.Digester.endElement(Digester.java:1128) ~[commons-digester3-3.2.jar:3.2]
	... 41 more

java.lang.RuntimeException: org.xml.sax.SAXParseException; lineNumber: 1; columnNumber: 840; Error at line 1 char 840: com.windowsazure.messaging.ApnsCredential.setkeyId(java.lang.String)

	at com.windowsazure.messaging.SyncCallback.failed(SyncCallback.java:53)
	at com.windowsazure.messaging.NamespaceManager.lambda$getNotificationHubAsync$0(NamespaceManager.java:74)
	at com.windowsazure.messaging.NotificationHubsService$1.completed(NotificationHubsService.java:73)
	at com.windowsazure.messaging.NotificationHubsService$1.completed(NotificationHubsService.java:63)
	at org.apache.hc.core5.concurrent.BasicFuture.completed(BasicFuture.java:123)
	at org.apache.hc.core5.concurrent.ComplexFuture.completed(ComplexFuture.java:72)
	at org.apache.hc.client5.http.impl.async.InternalAbstractHttpAsyncClient$1$1.completed(InternalAbstractHttpAsyncClient.java:280)
	at org.apache.hc.core5.http.nio.support.AbstractAsyncResponseConsumer$1.completed(AbstractAsyncResponseConsumer.java:101)
	at org.apache.hc.core5.http.nio.entity.AbstractBinAsyncEntityConsumer.completed(AbstractBinAsyncEntityConsumer.java:84)
	at org.apache.hc.core5.http.nio.entity.AbstractBinDataConsumer.streamEnd(AbstractBinDataConsumer.java:81)
	at org.apache.hc.core5.http.nio.support.AbstractAsyncResponseConsumer.streamEnd(AbstractAsyncResponseConsumer.java:142)
	at org.apache.hc.client5.http.impl.async.H2AsyncMainClientExec$1.streamEnd(H2AsyncMainClientExec.java:189)
	at org.apache.hc.core5.http2.impl.nio.ClientH2StreamHandler.consumeData(ClientH2StreamHandler.java:239)
	at org.apache.hc.core5.http2.impl.nio.AbstractH2StreamMultiplexer$H2Stream.consumeData(AbstractH2StreamMultiplexer.java:1636)
	at org.apache.hc.core5.http2.impl.nio.AbstractH2StreamMultiplexer.consumeDataFrame(AbstractH2StreamMultiplexer.java:1045)
	at org.apache.hc.core5.http2.impl.nio.AbstractH2StreamMultiplexer.consumeFrame(AbstractH2StreamMultiplexer.java:734)
	at org.apache.hc.core5.http2.impl.nio.AbstractH2StreamMultiplexer.onInput(AbstractH2StreamMultiplexer.java:445)
	at org.apache.hc.core5.http2.impl.nio.AbstractH2IOEventHandler.inputReady(AbstractH2IOEventHandler.java:65)
	at org.apache.hc.core5.http2.impl.nio.ClientH2IOEventHandler.inputReady(ClientH2IOEventHandler.java:39)
	at org.apache.hc.core5.reactor.ssl.SSLIOSession.decryptData(SSLIOSession.java:600)
	at org.apache.hc.core5.reactor.ssl.SSLIOSession.access$200(SSLIOSession.java:74)
	at org.apache.hc.core5.reactor.ssl.SSLIOSession$1.inputReady(SSLIOSession.java:202)
	at org.apache.hc.core5.reactor.InternalDataChannel.onIOEvent(InternalDataChannel.java:142)
	at org.apache.hc.core5.reactor.InternalChannel.handleIOEvent(InternalChannel.java:51)
	at org.apache.hc.core5.reactor.SingleCoreIOReactor.processEvents(SingleCoreIOReactor.java:178)
	at org.apache.hc.core5.reactor.SingleCoreIOReactor.doExecute(SingleCoreIOReactor.java:127)
	at org.apache.hc.core5.reactor.AbstractSingleCoreIOReactor.execute(AbstractSingleCoreIOReactor.java:86)
	at org.apache.hc.core5.reactor.IOReactorWorker.run(IOReactorWorker.java:44)
	at java.lang.Thread.run(Thread.java:750)
Caused by: org.xml.sax.SAXParseException; lineNumber: 1; columnNumber: 840; Error at line 1 char 840: com.windowsazure.messaging.ApnsCredential.setkeyId(java.lang.String)
	at org.apache.commons.digester3.Digester.createSAXException(Digester.java:3118)
	at org.apache.commons.digester3.Digester.createSAXException(Digester.java:3146)
	at org.apache.commons.digester3.Digester.endElement(Digester.java:1133)
	at com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.endElement(AbstractSAXParser.java:609)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanEndElement(XMLDocumentFragmentScannerImpl.java:1781)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl$FragmentContentDriver.next(XMLDocumentFragmentScannerImpl.java:2966)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl.next(XMLDocumentScannerImpl.java:601)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanDocument(XMLDocumentFragmentScannerImpl.java:504)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:841)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:770)
	at com.sun.org.apache.xerces.internal.parsers.XMLParser.parse(XMLParser.java:141)
	at com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.parse(AbstractSAXParser.java:1213)
	at com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl$JAXPSAXParser.parse(SAXParserImpl.java:642)
	at org.apache.commons.digester3.Digester.parse(Digester.java:1642)
	at org.apache.commons.digester3.Digester.parse(Digester.java:1701)
	at com.windowsazure.messaging.NotificationHubDescription.parseOne(NotificationHubDescription.java:170)
	at com.windowsazure.messaging.NamespaceManager.lambda$getNotificationHubAsync$0(NamespaceManager.java:72)
	... 27 more
Caused by: java.lang.NoSuchMethodException: com.windowsazure.messaging.ApnsCredential.setkeyId(java.lang.String)
	at java.lang.Class.getMethod(Class.java:1786)
	at com.windowsazure.messaging.PnsCredential.setProperty(PnsCredential.java:23)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.apache.commons.beanutils.MethodUtils.invokeMethod(MethodUtils.java:278)
	at org.apache.commons.digester3.CallMethodRule.end(CallMethodRule.java:500)
	at org.apache.commons.digester3.Digester.endElement(Digester.java:1128)
	... 41 more
```

**To Reproduce**
Try to get a notification hub by the hub path.

***Code Snippet***
``` 
                String connectionString = "Endpoint=sb://xxx.servicebus.windows.net/;SharedAccessKeyName=DefaultFullSharedAccessSignature;SharedAccessKey=xxx";
                String hubName = "xxx";

                NamespaceManager namespaceManager = new NamespaceManager(connectionString);
                NotificationHubDescription hubInfo = namespaceManager.getNotificationHub(hubName); // error occurs
```

**Expected behavior**
NotificationHubDescription is retrieved.

**Setup (please complete the following information):**
- Java version: 1.8.0_332
- Notification-Hubs-java-sdk.1.0.4

**Information Checklist**
Kindly make sure that you have added all the following information above and checkoff the required fields otherwise we will treat the issue as an incomplete report.  Please do not provide any private information in this bug report.

- [x] Bug Description Added
- [x] Repro Steps Added
- [x] Setup information Added

**Comment by zhangeugenia** [comment]
(reopening issue until new SDK version is published with the fix, sorry for the confusion. ETA will be sometime the week of Aug 16, 2023)

**Comment by zhangeugenia** [comment]
Fixed in version [1.0.5](https://github.com/Azure/azure-notificationhubs-java-backend/releases/tag/1.0.5)
